fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergAlterTableExecutor.java

```diff
@@ -405,4 +412,87 @@ public Void visitDropTagClause(DropTagClause clause, ConnectContext context) {
         return null;
     }
 
+    @Override
+    public Void visitAlterTableOperationClause(AlterTableOperationClause clause, ConnectContext context) {
+        IcebergTableOperation op = IcebergTableOperation.fromString(clause.getTableOperationName());
+        if (op == IcebergTableOperation.UNKNOWN) {
```

Comment:
I think this check can do in the analyzer stage

Reply:
this operation will be used for different catalog. So it may be more appropriate to check in their respective metadata.